WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewView.swift

```diff
@@ -259,6 +298,89 @@ private extension ProductDetailPreviewView {
     }
 }
 
+// MARK: - Package photo view
+
+private extension ProductDetailPreviewView {
+    struct PackagePhotoView: View {
```

Comment:
❓ I see that this view is quite identical to the one on the starting page. Should we reuse it and add an option to show/hide the subtitle and the Replace photo option instead?

Reply:
Good suggestion. Done in https://github.com/woocommerce/woocommerce-ios/pull/13255/commits/2c4bf7a80644dbfac0b742d2069ac05fd303084c and https://github.com/woocommerce/woocommerce-ios/pull/13255/commits/7524f0f5f962872bfcd1749b754cedba7bae8f1d.